app/posts/apply-for-teacher-training/2022-11-07-changing-how-we-ask-about-disabilities-and-health-conditions.md

```diff
@@ -0,0 +1,70 @@
+
+In addition, for the 2022 to 2023 recruitment cycle, the Higher Education Statistics Authority updated their recommended list of options to present.
+
+## What we changed
+
+![Screenshot showing page with the question ’Do you have any of these disabilities or health conditions?’](disabilities-and-health-conditions.png "Updated design for disabilities or health conditions question")
+
+We removed the first question, and instead present all users with a list of checkboxes.
+
+We changed the question to ‘Do you have any of these disabilities or health conditions?’ and added a ‘I do not have any of these disabilities or health conditions’ option.
+
+We added some content at the top of the page to confirm that a candidate’s answer is not shown to providers when they are deciding whether or not to offer a place on a course, and instead is only revealed if an offer has been accepted.
+
+We also added a link back to the section of the application where candidates can ask for support if they are disabled as it is important that users do not expect providers to consider their answer to this question at the interview or assessment stage.
+
+We reviewed the content for each of the options, with the aim to front-load the most recognisable words and phrases, and to avoid using hint text except to give examples.
+
+## Alternatives we considered
+
+The Higher Education Statistics Authority has some [guidance on collecting type of disability for initial teacher training] which includes suggested question wording from [Advance HE](https://www.advance-he.ac.uk). Their suggesting is to include a lengthy explanation of disability as defined in the Equality Act 2010, and then to ask this question:
+
+> Considering the above, do you have an impairment, health condition or learning difference that has substantial and long-term impact on your ability to carry out normal day-to-day activities?
+
+We considered that this was too much of a cognitive burden on candidates, particularly at the application stage of the process.
+
+The [equality information pattern] in the GOV.UK Design System asks an initial question of ‘Do you have any physical or mental health conditions or illness lasting or expected to last 12 months or more?’ and then a follow-up question of ‘Do any of your conditions or illnesses reduce your ability to carry out day to day activities?’. This pattern does not include a list of disability or health conditions.
+
+## Initial results
+
+Comparing the first 28 of the 2022 to 2023 recruitment cycle to the first 28 days of the previous cycle, 3 times more candidates have selected one or more disability or health condition (15% compared to 4%).
```

Comment:
I don't think you need to both talk about '3 times more' and give the percentages - I'd just give the percentages first.